command/node_status.go

```diff
@@ -386,6 +388,38 @@ func (c *NodeStatusCommand) formatNode(client *api.Client, node *api.Node) int {
 
 }
 
+func (c *NodeStatusCommand) outputNodeStatusEvents(node *api.Node) {
+	if !c.verbose {
```

Comment:
Sorry should have been clearer. The "details" column should only be outputted in verbose mode.